tests/e2e-playwright/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Targets for DEVELOPMENT of tests/public-api
+#
```

Comment:
I have a few questions:
1. What is the plan to have this running in the CI?
2. Where will the image with all the tooling be installed?
3. Are we sure we want to keep packaging the tests with the code?
4. How do we handle dependencies between the docker image and the dependencies required here

Reply:
So I am going step by step (my long-term plan is to have a process how to automatically generate test pipelines for each deployment we are running, I have ideas how to do it)

For now (this morning), I have just replaced the old puppeteer sim4life test with this new one. It is already running in the CI: https://git.speag.com/oSparc/e2e-testing/-/pipelines?page=1&scope=all&ref=production_zmt_aws (you can check the yml file and see how it is running. It is using their recommended image)

If you want to run/debug this test also see the nice headed mode you can do it anytime locally (as only requirement is to have installed python, you can for example run `make install-dev` and you are good to go, it needs just pytest-playwright library). 

If you want to run it in the headless mode via docker just run it the same way as you can see it in the UI. 

If you want to run in the headed mode via docker, that is more complicated and you would need to investigate what ports to allow, etc...